.chloggen/add_inspect_metrics.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
appears the breaking change was lost

Reply:
It was updated after [this comment](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/21803#discussion_r1332337698). Let's agree on moving it back to breaking or modifying the api version check on the comment bellow if that's ok.